builtin/settingtypes.txt

```diff
@@ -466,7 +441,30 @@ enable_raytraced_culling (Enable Raytraced Culling) bool true
 
 
 
-[*Shaders]
+[*Effects]
+
+#    Allows liquids to be translucent.
+translucent_liquids (Translucent liquids) bool true
+
+#    Leaves style:
+#    -   Fancy:  all faces visible
+#    -   Simple: only outer faces, if defined special_tiles are used
```

Comment:
this is an implementation detail
```suggestion
#    -   Simple: only outer faces
```

Reply:
FWIW this is documented under the drawtype `allfaces_optional`, hence it's safe to omit here.